source/_integrations/schlage.markdown

```diff
@@ -37,6 +37,12 @@ Once you have enabled the Schlage integration, you should see the following bina
 
 - **Keypad disabled** - Indicates that the keypad has been disabled, typically due to too many incorrect lock codes being attempted.
 
+## Select
+
+Once you have enabled the Schlage integration, you should see the following selects:
+
+- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely.
```

Comment:
```suggestion
- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely. For example, `0`: auto-lock is disabled, `15`: auto-lock after 15 seconds, `300`: auto-lock after 5 minutes.
```

Reply:
This doesn't hurt, but the user will see all the options once they drill down into the `select` too. Thanks for making the change